source/_components/utility_meter.markdown

```diff
@@ -184,3 +184,21 @@ utility_meter:
     source: sensor.gas_consumption
     cycle: monthly
 ```
+
+Additionally, you can add template sensors to compute daily and monthly total usage. 
+
+```yaml
```

Comment:
Since the YAML template contains Jinja, this causes issues with Jekyll.
Escape it using `{% raw %}`, e.g.:

````
{% raw %}
```yaml
sensor:
```
{% endraw %}
````

Reply:
> Since the YAML template contains Jinja, this causes issues with Jekyll.
> Escape it using `{% raw %}`, e.g.:
> 
> ```
> {% raw %}
> ```yaml
> sensor:
> ```
> {% endraw %}
> ```

Thanks. That worked. Learned something today. 😀